Example/Source/Examples/Interactive/InvalidateHeightViewController.swift

```diff
@@ -55,12 +53,15 @@ class InvalidateHeightViewController: BrickViewController, HasTitle {
     }
 
     @objc func toggleHeights() {
+        let newHeight: BrickDimension
         switch brick.height {
-        case .fixed(_): brick.height = .auto(estimate: .fixed(size: 100))
-        default: brick.height = .fixed(size: 200)
+        case .fixed(_): newHeight = .auto(estimate: .fixed(size: 100))
+        default: newHeight = .fixed(size: 200)
         }
+        
+        let size = BrickSize(width: .ratio(ratio: 1), height: newHeight)
+        self.brickCollectionView.resizeBrick(with: BrickIdentifiers.titleLabel, newSize: size)
```

Comment:
I think the idea is that its already implicitly animated since it delegates to performUpdates function which also does this. wrapping it in an animation block doesn't have any effect if you just want the default animation parameters